src/compute-client/src/plan/join/linear_join.rs

```diff
@@ -309,8 +304,8 @@ impl LinearJoinPlan {
         // Form and return the complete join plan.
         let plan = LinearJoinPlan {
             source_relation,
-            source_key,
-            initial_closure,
+            source_key: source_arrangement.as_ref().map(|(key, _, _)| key.clone()),
```

Comment:
I think the `.as_ref()` here is redundant.

Reply:
Fixed thanks.